Sources/SwiftInspectorCore/Tests/PropertyAnalzyerSpec.swift

```diff
@@ -357,12 +384,33 @@ final class PropertyAnalyzerSpec: QuickSpec {
         it("detects the property with no type information") {
           let result = try? sut.analyze(fileURL: fileURL)
           expect(result?.properties) == [
-            TypeWithPropInfo.PropInfo(
+            TypeProperties.PropertyData(
+              name: "thing",
+              typeAnnotation: nil,
+              comment: "",
+              modifier: [.private, .static])
+          ]
+        }
+      }
+
+      context("when there is a property attribute") {
```

Comment:
This looks good for now. Comforting to know we handle the case even if we don't pull out the attribute itself yet.